.github/workflows/localization_branch_sync.yml

```diff
@@ -0,0 +1,32 @@
+name: Sync Localization Branch
+on:
+  pull_request:
+    branches:
+      - 'main'
+    types: [closed]
+
+jobs:
+  delete_branch:
+    name: 'Delete Localization branch'
+    runs-on: ubuntu-latest
+    steps:
+    - uses: dawidd6/action-delete-branch@v3
+      name: 'delete'
+      if: ${{ github.event.pull_request.merged == true && github.event.pull_request.user.login == 'github-actions[bot]' && contains(github.event.pull_request.labels.*.names, 'localization_bot') }}
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Secrets will not be available in PRs that come from forks, have you considered this?

Reply:
Oh I was not aware of that..
Although this is only for the Bot created in our other github action which I believe does not come from a fork so we should be okay here?

Reply:
yes, if the PR comes form a fork we will have an error. it will say that the token is empty, which is not very intuitive. Can you add a check and if secrets.GITHUB_TOKEN == "" print some message, like 'Action cannot be executed on forks" Or something of the kind?